lib/grpc_web/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module GRPCWeb
-  VERSION = '1.2.1'
+  VERSION = '1.2.2'
```

Comment:
You don't need to (shouldn't) manually update this. It will be managed by Semantic Release Bot based on the PR and title. Check out [CONTRIBUTING.md](https://github.com/Gusto/grpc-web-ruby/blob/master/CONTRIBUTING.md) for more info.